src/govuk/i18n.mjs

```diff
@@ -294,6 +294,7 @@ I18n.pluralRulesMap = {
  * @type {Object<string, function(number): PluralRule>}
  */
 I18n.pluralRules = {
+  /* eslint-disable jsdoc/require-jsdoc */
```

Comment:
Great call there! 🙌🏻

Reply:
Was a bit much to add JSDoc to all of those 😅

Reply:
At that point who's counting 🤣